src/test/java/teammates/test/cases/datatransfer/AdminEmailAttributesTest.java

```diff
@@ -0,0 +1,140 @@
+package teammates.test.cases.datatransfer;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.google.appengine.api.datastore.Text;
+
+import teammates.common.datatransfer.attributes.AdminEmailAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.FieldValidator;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.StringHelper;
+import teammates.common.util.TimeHelper;
+import teammates.storage.entity.AdminEmail;
+import teammates.test.driver.StringHelperExtension;
+
+/**
+ * SUT: {@link AdminEmailAttributes}.
+ */
+public class AdminEmailAttributesTest extends BaseAttributesTest {
+
+    private AdminEmailAttributes adminEmailAttributes;
+
+    @BeforeClass
+    public void classSetup() {
+        List<String> addressReceiverListString = Arrays.asList("example1@test.com", "example2@test.com");
+        List<String> groupReceiverListFileKey = Arrays.asList("listfilekey", "listfilekey");
+        String validSubject = "subject of email";
+        Text content = new Text("valid email content");
+
+        adminEmailAttributes = new AdminEmailAttributes(
+                validSubject, addressReceiverListString, groupReceiverListFileKey, content, new Date());
+
+        adminEmailAttributes.createDate = new Date();
+    }
+
+    @Test
+    public void testValidate() throws Exception {
+        assertTrue("Valid input", adminEmailAttributes.isValid());
+
+        List<String> actual = adminEmailAttributes.getInvalidityInfo();
+        assertTrue("Valid input should return an empty list", actual.isEmpty());
```

Comment:
I've seen other tests written something similar to "... an empty _string_".
Perhaps  "... an empty list of _errors_" would be better here?

I'll go with _errors_ for now.